### PR TITLE
Baitap2

### PR DESCRIPTION
**Pull request by at-thongtran**
Hi anh,

Em xin nộp bài tập html buổi 1.
Anh có thể review dùm em.

Thanks anh.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html>
+  <head>
```
Comment (at-lainguyen) [inline index.html]: Don't need to indent for `head`, `body` tags

File: index.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    <title>SPECIALIZED CLINIC</title>
+  </head>
+  <body>
+    <header>
+      <section class="logo-item">
+        <a href=""><img src="images/logo.jpg"></a>
+        <div class="logo-content">
+          <h1>SPECIALIZED CLINIC</h1>
+          <p>Physiotherapy - Rehabilitation</p>
```
Comment (at-lainguyen) [inline index.html]: Consider to use `span` tag, `p` is paragraph.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SPECIALIZED CLINIC</title>
+  </head>
+  <body>
+    <header>
+      <section class="logo-item">
+        <a href=""><img src="images/logo.jpg"></a>
+        <div class="logo-content">
```
Comment (at-lainguyen) [inline index.html]: If write HTML like this, that mean we can't click on this text to redirect to home. Consider to wrap it by `a` tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+          <h1>SPECIALIZED CLINIC</h1>
+          <p>Physiotherapy - Rehabilitation</p>
+        </div>
+      </section>
+      <section class="contract-item">
```
Comment (at-lainguyen) [inline index.html]: Don't use section in some case like this. Section should have title

File: index.html [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+        <i class="fa fa-phone"></i>
```
Comment (at-lainguyen) [inline index.html]: Consider to use `span` tag for icon. Only use `i` tag if you have icon fonts or already style for that tag.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+          <p>Physiotherapy - Rehabilitation</p>
+        </div>
+      </section>
+      <section class="contract-item">
+        <i class="fa fa-phone"></i> 
+        <p>Question for the doctor? <span>05113941 977</span></p>
+      </section>
+      <nav class="nav-items">
+        <ul class="nav-menu">
+          <li><a href="">HOME</a></li>
```
Comment (at-lainguyen) [inline index.html]: Consider to use lowercases for all text. HTML only use for file structure.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+          <li><a href="">CONSULT & CONTACT</a></li>
+        </ul>
+        <ul class="nav-social">
+          <li><a href=""><i class="fa fa-facebook"></i></a></li>
+          <li><a href=""><i class="fa fa-google"></i></a></li>
+          <li><a href=""><i class="fa fa-youtube"></i></a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
```
Comment (at-lainguyen) [inline index.html]: Where is class for `main` tag?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+      <section class="breadcrumb">
+        <ul class="breadcrumb-list">
+          <li class="breadcrumb-item"><a href="">HOME</a></li>
+          <span>/</span>
+          <li class="breadcrumb-item"><a href="">HEALTH SERVICES</a></li>
```
Comment (at-lainguyen) [inline index.html]: Last breadcrumbs don't have `a` tag. We can't click on last breadcrumbs

File: index.html [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+        </ul>
+      </section>
+      <section class="main-content">
+        <aside class="aside-left">
+          <ul class="menu-list">
+            <li class="menu-open-list">
+              <span class="open-list">ADUITS</span>
```
Comment (at-lainguyen) [inline index.html]: Mis-spelling: `aduits` => `adults`

Comment (at-lainguyen) [reply index.html]: Miss sub icon

File: index.html [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+                <li class="menu-open-items"><a href="">Knee joint, hip joint</a></li>
+                <li class="menu-open-items"><a href="">Footsore.</a></li>
+                <li class="menu-open-items"><a href="">Diseases of the muscles</a></li>
+                <li class="menu-open-items"><a href="">Peripheral newropathy</a></li>
+                <li class="menu-open-items"><a href="">Sequelae of stroke</a></li>
+                <li class="menu-open-items"><a href="">Traumatic sequelae</a></li>
+              </ul>
+            </li>
+            <li class="menu-open-list">
+              <span class="open-list">CHILDREN</span>
```
Comment (at-lainguyen) [inline index.html]: miss add icon

File: index.html [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+            </li>
+          </ul>
+          <form class="book-schedule" method="post" action="#">
+            <title>BOOK SCHEDULE</title>
```
Comment (at-lainguyen) [inline index.html]: Why `title`?

File: index.html [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+            <input type="text" name="fulllname" placeholder="Full Name">
+            <input type="text" name="address" placeholder="Address">
+            <input type="text" name="phone-number" placeholder="Phone Number">
+            <div id="time-item">
+              <input type="text" name="time" placeholder="Time">
+              <i class="fa fa-time"></i>
+            </div>
+            <button type="submit"> SEND </button>
```
Comment (at-lainguyen) [inline index.html]: remove empty space for text `SEND`

File: index.html [hunk 13]
```diff
@@ -0,0 +1,180 @@
...
+              <i class="fa fa-time"></i>
+            </div>
+            <button type="submit"> SEND </button>	
+          </form>
+        </aside>
+        <aside class="aside-right">
+          <ul class="healthy-servics-list">
+            <li class="healthy-servics-item">
+              <a href=""><img src="images/1.jpg"></a>
+              <div class="healthy-servics-content">
```
Comment (at-lainguyen) [inline index.html]: miss more detail icon

Comment (at-lainguyen) [inline index.html]: consider about mis-spelling

File: index.html [hunk 14]
```diff
@@ -0,0 +1,180 @@
...
+              <a href=""><img src="images/5.jpg"></a>
+              <div class="healthy-servics-content">
+                <h3>NECK SHOULDER PAIN</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna</p>
+              </div>
+            </li>
+            <li class="healthy-servics-item">
+              <a href=""><img src="images/6.jpg"></a>
+              <div class="healthy-servics-content">
+                <h3>NECK SHOULDER PAIN</h3>
```
Comment (at-lainguyen) [inline index.html]: missing some SLOW SAY items, don't copy paste for all text

File: index.html [hunk 15]
```diff
@@ -0,0 +1,180 @@
...
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna</p>
+              </div>
+            </li>
+          </ul>
+        </aside>
+      </section>
+    </main>
+    <footer>
```
Comment (at-lainguyen) [inline index.html]: missing class for footer

File: index.html [hunk 16]
```diff
@@ -0,0 +1,180 @@
...
+      <section class="footer-content">
+        <ul class="footer-list">
+          <li class="fotter-item">
```
Comment (at-lainguyen) [inline index.html]: `fotter`?

File: index.html [hunk 17]
```diff
@@ -0,0 +1,180 @@
...
+          </li>
+        </ul>
+      </section>
+      <section class="footer-author">
+        <p>&copy; 2013 Copyright of specia_ized clinic</p>
+        <p>DESIGN BY <span><a href="">DD.COM</a></span></p>
+      </section>
+    </footer>
+  </body>
+</html>
```
Comment (at-lainguyen) [inline index.html]: missing end of file

File: index.html [hunk 18]
```diff
@@ -0,0 +1,180 @@
...
+            <ul>
+              <li><a href=""><i class="">FACEBOOK</i></a></li>
+              <li><a href=""><i class="">GOOGLE+</i></a></li>
+              <li><a href=""><i class="">YOUTUBE</i></a></li>
+            </ul>
+          </li>
+        </ul>
+      </section>
+      <section class="footer-author">
+        <p>&copy; 2013 Copyright of specia_ized clinic</p>
```
Comment (at-lainguyen) [inline index.html]: Why it is the same with Thai and Luan?

Comment (at-thongtran) [reply index.html]: Xin lỗi anh, em có copy dòng đó

**Pull request closed without merging**